scss/_config.scss

```diff
@@ -40,7 +40,13 @@
 //);
 
 // vertical spacing
-//$vertical_spacing: (
-//        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
-//        "md":    0 1 2 4 6 8 12 16 24 30 40 50 60 70,
-//);
\ No newline at end of file
+$vertical_spacing: (
+        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
+        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
+);
+
+// loading class
+$loading_class: (
+        "enable": false,
+        "isWoocomerce": true // true if project using woocommerce
```

Comment:
The default value of `enable` should be `true`. The default value of `isWoocommerce` should be `false`.